tests/tests_integration/test_api/test_geospatial.py

```diff
@@ -0,0 +1,410 @@
+import random
+import sys
+import time
+import uuid
+
+import pytest
+from cognite.client import utils
+from cognite.client.exceptions import CogniteAPIError
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes.geospatial import (
+    CoordinateReferenceSystem,
+    Feature,
+    FeatureList,
+    FeatureType,
+    FeatureTypeUpdate,
+    OrderSpec,
+    PropertyAndSearchSpec,
+)
+
+COGNITE_CLIENT = CogniteClient(max_workers=1)
```

Comment:
Should use the `cognite_client` fixture instead. This will not always instantiate auth handling correctly.
```suggestion
```

Reply:
Ok, I will use it.